### DISCUSSION TITLE
Allow images to be attached to a block

### DISCUSSION THREAD

**Discussion by lumendescomp** [tone-row/flowchart-fun#388] [Features / Syntax]
The ideia involves being able to associate an image to a block (just like a comment in notion, where you can click it and new info will pop) enabling it to be clicked whenever you wanna see the extra info. For example: I'm writing about the purpose of some JavaScript code in a block, I'd like to attach an image of this code instead of typing it.

**Comment by rob-gordon** [comment]
@lumendescomp When you say enabled when clicking / do you mean that the node expands to show it, or that a popup/tooltip appears with the image?

I have future plans of adding the ability to have image nodes, would that solve the problem you're having?
(Thanks for opening a discussion!)

**Reply by lumendescomp** [reply]
@rob-gordon I guess it does solve the problem, however for visualization/organization purposes I think a popup/tooltip handles it better.

**Reply by rob-gordon** [reply]
Cool. I appreciate the input! I will think about how I could add that I feature.
